includes/class-controller.php

```diff
@@ -49,8 +51,38 @@ private function api_rewrite() {
 		}
 	}
 
+	/**
+	 * Ajax action to clear the Log file.
+	 *
+	 * @return void
+	 */
+	public function clear_log() {
+		if ( isset( $_POST['nonce'] ) || wp_verify_nonce( $_POST['nonce'], 'aspireupdate-ajax' ) ) {
```

Comment:
```suggestion
		if ( isset( $_POST['nonce'] ) && wp_verify_nonce( $_POST['nonce'], 'aspireupdate-ajax' ) ) {
```
This should use `&&` rather than `||`.

🔢 Applies elsewhere in the PR.